### ISSUE TITLE
cos(x)

### ISSUE THREAD

**Issue by KirillOsenkov** [KirillOsenkov/QuickInfo#3]

**Comment by VladimirRybalko** [comment]
Hi @KirillOsenkov 
Seems **cos** function is working now.
Look at this: http://quickinfo.io/?cos(3.14)

**Comment by KirillOsenkov** [comment]
Yeah, I know :) it's just when there's an identifier passed as an argument. This is actually a bug in https://github.com/KirillOsenkov/MathParser, I'm just too lazy to fix it :) I've also been using this bug to test the exception reporting.
